polkadot-parachains/statemine/src/constants.rs

```diff
@@ -14,19 +14,20 @@
 // limitations under the License.
 
 pub mod currency {
+	use kusama_runtime_constants as constants;
 	use node_primitives::Balance;
 
 	/// The existential deposit. Set to 1/10 of its parent Relay Chain (v9020).
```

Comment:
@Doordashcon Just noticed that we'll want to remove the reference to a specific runtime version.
```suggestion
	/// The existential deposit. Set to 1/10 of its parent Relay Chain.
```